src/CommandingOfficers/AWBW/BM/OlafDSBW.java

```diff
@@ -0,0 +1,137 @@
+package CommandingOfficers.AWBW.BM;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.*;
+import CommandingOfficers.AWBW.AWBWCommander;
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.GlobalWeatherEvent;
+import Engine.GameEvents.MassDamageEvent;
+import UI.UIUtils;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Terrain.Environment.Weathers;
+import Units.Unit;
+import Units.UnitContext;
+
+public class OlafDSBW extends AWBWCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Olaf", UIUtils.SourceGames.AWBW, UIUtils.BM, "3BW");
+      infoPages.add(new InfoPage(
+            "AWDS Olaf for AWBW\n"
+          + "Olaf version that gets +attack for 2 turns instead of slowing people for his powers.\n"
```

Comment:
If this is the actual passive from AWDS, I'd probably just quote the AWDS text here instead of declaring this as an alternate "version", since that makes it sound like this isn't drawn from the cart.